app/config/settings.py

```diff
@@ -128,6 +129,11 @@ def strtobool(val) -> bool:
 # About Flatpage
 FLATPAGE_ABOUT_URL = os.environ.get("FLATPAGE_ABOUT_URL", "/about/")
 
+# Costs
+AWS_FILE_STORAGE_COSTS = os.environ.get("AWS_FILE_STORAGE_COSTS", 0.00004)
+AWS_DOCKER_STORAGE_COSTS = os.environ.get("AWS_DOCKER_STORAGE_COSTS", 0.04)
+AWS_COMPUTE_COSTS = os.environ.get("AWS_COMPUTE_COSTS", 0.3)
```

Comment:
The `AWS_` namespace is reserved so maybe use `CHALLENGES_` instead (the costs also include non-aws things too such as off site backups). It is a good idea to add units to the names of the variables, and if you are doing things with currency then it is best to use integer arithmetic with (euro) cents as the unit, so `_CENTS_PER_GB_PER_MONTH` etc.